src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Language/Legacy/ParserTestBase.cs

```diff
@@ -175,44 +172,19 @@ private static string NormalizeNewLines(string content)
             return Regex.Replace(content, "(?<!\r)\n", "\r\n", RegexOptions.None, TimeSpan.FromSeconds(10));
         }
 
-        internal virtual void BaselineTest(RazorSyntaxTree syntaxTree, bool verifySyntaxTree = true)
+        internal virtual void BaselineTest(RazorSyntaxTree syntaxTree, bool verifySyntaxTree = true, bool ensureFullFidelity = true)
         {
             if (verifySyntaxTree)
             {
-                SyntaxTreeVerifier.Verify(syntaxTree);
+                SyntaxTreeVerifier.Verify(syntaxTree, ensureFullFidelity);
```

Comment:
Are there still tests that set this to false? IMO it should be really hard to opt out of this, unless you're staging the rollout.

Reply:
> unless you're staging the rollout

What does that mean?